claim-check/code-samples/sample-4/client-producer/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Configuration;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace client_producer
+{
+    class Program
+    {
+        public static PayloadDetails payloadDetails = new PayloadDetails();
```

Comment:
This variable should not be here.  See below changes for the SendNotification method